openApiDocumentation.yml

```diff
@@ -0,0 +1,561 @@
+        400:
+          description: Invalid request
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/BadRequestError'            
+                    
+        401:
+          description: unauthorized
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/UnauthorizedError'
+        500:
+          description: Internal Server Error
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/InternalServerError'
+                  
+  /v1/customers/forgot-password:
+    get:
+        summary: email exists
+        description: Check if an email address exists in the database
+        tags: 
+          - Forgotpassword Journey
+        parameters:
+          - in: query
+            name: emailid
+            required: true
+            schema:
+              type: string
+              format: email
+              example: user@example.com
+        responses:
+          200:
+            description: Email exists
+            content:
+              application/vnd.api+json:
+                schema:
+                  type: object
+                  properties:
+                    token:
+                      type: string
+                      example: "asdfghjkwertyuiopsdfghjkexrctybujizwexrtyhuxecrvtyhus3d4f5gtyhud4f5g6h7j8edrfg6h78d5fg6h78fg6h78f5g6h7j8df5g6h7j8frtgyftghfg678g5f6h78fgyed8ijed8ujefujid8ifed8uhedyuhedujeducheidw9se"
+                    otp:
+                      type: string
+                      example: "342988"
+                    
+          404:
+            description: Email not found
+            content:
+              application/vnd.api+json:
+                schema:
+                  $ref: '#/components/schemas/NotFoundError'  
+          500:
+            description: Internal Server Error 
+            content:
+              application/vnd.api+json:
+                schema:
+                  $ref: '#/components/schemas/InternalServerError'        
+  /v1/customers/reset-password:
+      patch:
+        summary: create new password
+        description:  Reset password for a given account number
+        tags: 
+          - Forgotpassword Journey
+        requestBody:
+          description: new password and re-enter password
+          content:
+            application/vnd.api+json:
+              schema:
+                
+                
+                properties:
+                  emailId:
+                    type: string
+                    description: EmailId
+                  password:
+                    type: string
+                    description: password
+                  re-enter password:
+                    type: string
+                    description: Re-enter password
+                    format: password
+        responses:
+          201:
```

Comment:
201 is used for creating...are you doing password creation or password updation?

Reply:
after updating the password you have to send back the token as response to frontend. Then only they can use the token in the next api calls to fetch the details for dashboard. I have already asked you to change this